docs/blog/rendered/ibis-version-6.1.0-release.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>table without rows for <code>False</code>.</blockquote><p>This is only true in the inner join case. The <code>LEFT</code>,<code>RIGHT</code>,and <code>FULL</code> outer join cases can all potentially return rows.       </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>